firefox_puppeteer/api/keys.py

```diff
@@ -1,20 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this file,
-# You can obtain one at http://mozilla.org/MPL/2.0/.
-
-import marionette
-
-
-class Keys(marionette.keys.Keys):
```

Comment:
But a neighboring library won't have self.browser, and I made use of this functionality in a toolbars class.